gulpfile.js

```diff
@@ -259,6 +263,15 @@ function cssWatch(cb) {
     cb();
 }
 
+/**
+ * Transpile ES6 JavaScript files.
+ */
+function transpileJs() {
+    return gulp.src('media/js/components/*.es6.js')
+        .pipe(babel({presets: ['@babel/preset-env']}))
+        .pipe(gulp.dest('media/js/compiled'))
```

Comment:
Instead of checking-in compiled JS into GitHub, we may want follow how Sass & Less assets are compiled and bundled as convention:

1. Transpile JS files into `static_build`
2. The content of `static_build` is then bundled and minified into `static_final` ready for consumption.